### PR TITLE
[ws-man-bridge] Remove EE

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

No longer needed to have the seperation

## Related Issue(s)

Fixes #

## How to test

ws man bridge starts up in preview

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - mp-ws-man-d2a7bd648e</li>
	<li><b>🔗 URL</b> - <a href="https://mp-ws-man-d2a7bd648e.preview.gitpod-dev.com/workspaces" target="_blank">mp-ws-man-d2a7bd648e.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

**Comment by easyCZ** [comment]
```
ws-manager-bridge-5866677f46-vlkb4   2/2     Running   0             19m
```

File: components/ws-manager-bridge/src/prebuild-updater-db.ts [hunk 1]
```diff
@@ -9,16 +9,15 @@ import { TraceContext } from "@gitpod/gitpod-protocol/lib/util/tracing";
...
 import { WorkspaceDB } from "@gitpod/gitpod-db/lib/workspace-db";
-import { MessageBusIntegration } from "../../src/messagebus-integration";
-import { PrometheusMetricsExporter } from "../../src/prometheus-metrics-exporter";
-import { filterStatus } from "../../src/bridge";
+import { MessageBusIntegration } from "./messagebus-integration";
+import { PrometheusMetricsExporter } from "./prometheus-metrics-exporter";
+import { filterStatus } from "./bridge";
 
 @injectable()
-export class PrebuildUpdaterDB implements PrebuildUpdater {
```
Comment (svenefftinge) [inline components/ws-manager-bridge/src/prebuild-updater-db.ts:21]: nit: Would be nice class to rename this to PrebuildUpdater

**Review by svenefftinge — APPROVED** [review]

File: components/ws-manager-bridge/src/prebuild-updater-db.ts (hunk 1 above)
Comment (easyCZ) [reply components/ws-manager-bridge/src/prebuild-updater-db.ts]: done

File: components/ws-manager-bridge/src/prebuild-updater.ts [hunk 2]
```diff
@@ -4,22 +4,126 @@
...
+import { PrometheusMetricsExporter } from "./prometheus-metrics-exporter";
+import { filterStatus } from "./bridge";
 
-export const PrebuildUpdater = Symbol("PrebuildUpdater");
+@injectable()
+export class PrebuildUpdater {
+    @inject(PrebuildStateMapper)
+    protected readonly prebuildStateMapper: PrebuildStateMapper;
 
-export interface PrebuildUpdater {
```
Comment (easyCZ) [inline components/ws-manager-bridge/src/prebuild-updater.ts:14]: diff is now weird that I renamed the original file, but effectively the only implementation of PrebuildUpdater is the one we had, and there's no interface anymore

**Pull request merged**